README.md

```diff
@@ -492,6 +492,9 @@ expect(actual).toNot(beIdenticalTo(expected))
 expect(actual) !== expected
 ```
 
+Its important to remember that beIdenticalTo only makes sense when comparing types with reference semantics, which have a notion of identity. In Swift, that means a `class`. This matcher will not work with types with value semantics such as `struct` or `enum`. If you need to compare two value types, you can either compare individual properties or if it makes sense to do so, make your type implement `Equatable` and use Nimble's equivalence matchers instead.
```

Comment:
Looks good! Do you think it makes sense to have "beIdenticalTo" as "`beIdenticalTo`"?